internal/backend/httpsvc/intercept.go

```diff
@@ -60,26 +60,29 @@ func intercept(z *zap.Logger, cfg *LoggerConfig, extractors []RequestLogExtracto
 	unlogged := regexp.MustCompile(strings.Join(cfg.UnloggedRegexes, `|`))
 
 	return func(w http.ResponseWriter, r *http.Request) {
-		if unlogged.MatchString(r.URL.Path) {
-			next.ServeHTTP(w, r)
-			return
-		}
-
 		w.Header().Set("X-AutoKitteh-Process-ID", fixtures.ProcessID())
 
 		l := z.With(zap.String("method", r.Method), zap.String("path", r.URL.Path))
 		rwi := &responseInterceptor{ResponseWriter: w, StatusCode: http.StatusOK, logger: l}
 
 		w.Header().Set("Trailer", "X-AutoKitteh-Duration")
 
+		// Call the next handler in the chain, and calculate the duration.
 		startTime := time.Now()
 		r = r.WithContext(context.WithValue(r.Context(), startTimeCtxKey, startTime))
 
 		next.ServeHTTP(rwi, r)
-		d := time.Since(startTime)
 
+		d := time.Since(startTime)
```

Comment:
What's d? Duration?
Better to have endTime

Reply:
We're logging the duration, not the end time, so having a variable for that is an unnecessary extra step.

The return type of `time.Since()` is `time.Duration` so `d` is a standard variable name. But I don't mind renaming to `duration` - done.